src/lib/extract/extractBrush.ts

```diff
@@ -40,6 +40,16 @@ export default function extractBrush(color?: Color) {
     return int32ARGBColor;
   }
 
+  // iOS PlatformColor
+  if ('semantic' in color) {
```

Comment:
I think here we should also add a check for if `color` is an object, as suggested here by @gaodeng :  https://github.com/oblador/react-native-svg/pull/1/files#diff-955398674a5278517eeeb9b5daca466f8e891b408d79757c3afbfa58a552a324R78